src/Workspaces/Core/Portable/ExternalAccess/UnitTesting/Api/UnitTestingProjectExtensions.cs

```diff
@@ -2,14 +2,16 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
+
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api
 {
     internal static class UnitTestingProjectExtensions
     {
-        public static string GetDebugName(this ProjectId projectId)
+        public static string? GetDebugName(this ProjectId projectId)
```

Comment:
Will this be a breaking change?

Reply:
No, nullable annotations are not breaking.